instrumentation/opentelemetry-instrumentation-grpc/tests/test_client_interceptor.py

```diff
@@ -169,7 +169,11 @@ def test_stream_stream(self):
             40, 40, "/GRPCTestServer/BidirectionalStreamingMethod"
         )
 
-    def _verify_error_records(self, method):
+    def _verify_error_records(
+        self, method, status_code=grpc.StatusCode.UNAVAILABLE
+    ):
+        # Updating grpc to 1.31.0 makes the status codes for two test cases
```

Comment:
i guess grpc `1.31.0` changed something in how connections are handled. the reason why the tests are now returning a different status code is that `self.channel` was not properly closed on `tearDown` (see #1027). the left over connection/channel of the previous test is then causing that the current test cannot connect to the test server which in turn leads to the status code `UNAVAILABLE`.
if just the failing test is run it should actually be green, or if failing test is removed the next test will suddenly fail.

Reply:
Thanks, @mariojonke. After the changes added in #1027 to properly close the channel, I can run the test cases with `tox -e py38-test-instrumentation-grpc` with `grpcio==1.31.0`. This PR does not seem necessary anymore, closing it.